client/agent_project_assignment.go

```diff
@@ -2,11 +2,11 @@ package client
 
 // Key is project id.
 // Value is agent id.
-type AssignProjectsAgentsAssignmentsPayload map[string]interface{}
+type AssignProjectsAgentsAssignmentsPayload map[string]string
 
 type ProjectsAgentsAssignments struct {
 	DefaultAgent   string                 `json:"defaultAgent"`
-	ProjectsAgents map[string]interface{} `json:"ProjectsAgents"`
+	ProjectsAgents map[string]interface{} `json:"projectsAgents"`
```

Comment:
nit

Could be changed to `map[string]interface{}` as well, right?
```suggestion
	ProjectsAgents map[string]string `json:"projectsAgents"`
```

Reply:
or "string" if it's always a string...

Reply:
Yeah, sorry, that's what I meant